internal/annotations/annotations.go

```diff
@@ -355,3 +356,9 @@ func ExtractUserTags(anns map[string]string) []string {
 	}
 	return strings.Split(val, ",")
 }
+
+// ExtractRewriteURI extracts the rewrite annotation value.
+func ExtractRewriteURI(anns map[string]string) (string, bool) {
+	s, ok := anns[AnnotationPrefix+RewriteURIKey]
+	return s, ok
```

Comment:
```suggestion
	return anns[AnnotationPrefix+RewriteURIKey]
```

Reply:
Here use `s, ok` is because we will do anything if annotation doesn't exist and use '/' as default if annotation exists but is empty.

Reply:
The detailed behaviour maybe need more discussion :).

Reply:
My bad, it's a map thus it tries to return only items when used directly in return. It's ok. About behavior, you can mention in the description